profiles/LudovicoSforza.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/44255157/153160232-39e415ad-7e93-4540-acdd-8b7e5d69a9a4.png",
+    "issueId": 1010,
+    "name": "Ludovico Sforza",
+    "username": "ludovicosforza"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```